rules/problem.yml

```diff
@@ -65,22 +65,22 @@ rules:
     formats:
     - oas3
     severity: hint
-    recommended: false
+    recommended: true
     # Search for 4xx, 5xx and default responses schema.
     given: >-
-      $.paths.[*].responses[?(@property && @property.match(/^(4|5|default)/)  && !@["x-noqa"] )][[schema]]
+      $..[responses][?(@property && @property.match(/^(4|5|default)/)  && !@["x-noqa"] )][[schema]].properties
     then:
       function: schema
       functionOptions:
         schema:
-          type: object
-          properties:
-            status:
-              type: integer
-            title:
-              type: string
-            detail:
-              type: string
+          anyOf:
```

Comment:
Is oneOf better or can a designer include any combination?

Reply:
I was told to avoid anyOf unless it's required, because it's more computationally intensive. Eg.

```
## anyOf
for item in anyOf:
  if target =~ item:
    return True
```

```
# oneOf
count = 0
for item in oneOf:
  if target =~ item:
    count += 1
  if count > 1:
    return False
return True
```

Reply:
Makes total sense.